src/example-types/metricGroups/pivot.js

```diff
@@ -97,52 +97,52 @@ let buildQuery = async (node, schema, getStats) => {
   // Don't consider deeper levels than +1 the current drilldown
   // This allows avoiding expansion until ready
   // Opt out with falsey drilldown
-  let groups = node.drilldown
-    ? _.take(_.size(node.drilldown) + 1, node.groups)
-    : node.groups
+  let rows = node.drilldown
+    ? _.take(_.size(node.drilldown) + 1, node.rows)
+    : node.rows
 
   let statsAggs = { aggs: aggsForValues(node.values, schema) }
-  // buildGroupQuery applied to a list of groups
-  let buildNestedGroupQuery = async (statsAggs, groups, groupingType, sort) => {
+  // buildGroupQuery applied to a list of rows
+  let buildNestedRowQuery = async (statsAggs, rows, groupingType, sort) => {
```

Comment:
This doesn’t actually make sense because it’s used to build queries for both rows and columns. Using the term group here is probably the right move - same for everything in this method.